DonjinKrawler-Common/src/main/java/krawlercommon/enemies/small/SmallZombie.java

```diff
@@ -20,4 +22,22 @@ public void update(EnemyStrategy strategy) {
     public Object deepCopy() throws CloneNotSupportedException {
         return null;
     }
+
+    public void setInterval() {
+        this.updateIntervalSeconds = 2;
+    }
+
+    public void setStrategies() {
+        this.strategies = Map.of(
+                Phases.AWAY, new MoveAwayFromPlayer(),
+                Phases.RANDOM, new MoveRandomly(),
+                Phases.TOWARDS, new MoveTowardPlayer(),
+                Phases.ATTACK, new Attack()
+        );
+    }
+
+    @Override
+    public void debug() {
+        System.out.println("Small zombie has been created");
```

Comment:
I presume these debug methods are called all the time, this really spams the logs with unnecessary stuff.

Reply:
Nu čia tik pakūrus serverį realiai tai susivykdo. Nelabai sugalvoju kokį nebūtina hooką būtų galima pritaikyti